src/redux/validation/validation.selectors.ts

```diff
@@ -67,6 +67,9 @@ export const problemsByResourcesSelector = createDeepEqualSelector(
 export const errorsByResourcesSelector = (state: ValidationState) => problemsByResourcesSelector(state, 'error');
 export const warningsByResourcesSelector = (state: ValidationState) => problemsByResourcesSelector(state, 'warning');
 
+export const errorsResourcesIdsSelector = (state: ValidationState) => Object.keys(errorsByResourcesSelector(state));
+export const warningsResourcesIdsSelector = (state: ValidationState) => Object.keys(warningsByResourcesSelector(state));
```

Comment:
Maybe it helps to have `createSelector` here as well for the memoization, or we could even use `createDeepSelector` 🤔 
```suggestion
export const errorsResourceIdsSelector = createSelector(errorsByResourcesSelector, errors => Object.keys(errors));
export const warningsResourcesIdsSelector = createSelector(warningsByResourcesSelector, warnings => Object.keys(warnings));

```